README.md

```diff
@@ -5,12 +5,24 @@ parameters in a global, hierarchical key database._
 
 <img src="https://cdn.rawgit.com/ElektraInitiative/libelektra/master/doc/images/circle.svg" alt="Elektra" width="50" />
 
-The core is a small library implemented in C. The plugin-based framework fulfills many
-configuration-related tasks to avoid any unnecessary code duplication
-across applications while it still allows the core to stay without any
-external dependency. Elektra abstracts from cross-platform-related issues
-with a consistent API, and allows applications to be aware of other
-applications' configurations, leveraging easy application integration.
+Elektra provides a simple API for applications to access and update configuration data. The
+configuration data comes from configuration files that are _mounted_ into Elektra similar to 
```

Comment:
This is a good example and important for getting an idea but not necessarily true. While Elektra favours the usage of configuration files (because of its resolvers and its current set of storage plugins), it actually also supports to integrate any other data sources as well.